src/main/kotlin/no/nav/hm/grunndata/register/series/SeriesRegistration.kt

```diff
@@ -127,7 +127,6 @@ data class SeriesRegistrationDTO(
     fun isPublishedProduct(): Boolean {
         return draftStatus == DraftStatus.DONE &&
             adminStatus == AdminStatus.APPROVED &&
-            status == SeriesStatus.ACTIVE &&
```

Comment:
Trur vi bør erstatte den som er sletta med: 

status != SeriesStatus.DELETED

for sikkerhets skuld